docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Keyacrd? (have no idea how to call it)
+version: 0.0.0
+status: Draft
+authors: Roman Volosovskyi <roman@status.im>
+redirect_from:
+  - /keycard.html
+---
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Multiaccount creation](#Multiaccount-creationrestoring)
+4. [Multiaccount restoring via pairing](#Multiaccount-restoring-via-pairing)
+5. [Multiaccount unlocking](#Multiaccount-unlocking)
+6. [Where are the keys stored?](#Where-are-the-keys-stored)
+
+## Abstract
+
+In this specification, we describe how Satus communicates with Keycard to create, store and use multiaccount.
+
+## Definitions
+
+
+| Term | Description |
+| -------- | ------------------ |
+|      |      |
+
+
+## Multiaccount creation/restoring
+### Creation and restoring via mnemonic
+1. `status-im.hardwallet.card/get-application-info` 
+    request: `nil`
+    response: `{"initialized?" false}`
+2. `status-im.hardwallet.card/init-card` 
+   request: `{:pin 123123}`
+   response: 
+   ```clojure
+   {"password" "nEJXqf6VWbqeC5oN", 
+    "puk" "411810112887", 
+    "pin" "123123"}
+   ```
+3. `status-im.hardwallet.card/get-application-info` 
+    request: `nil`
+    response: 
+    ```clojure
+    {"free-pairing-slots" 5, 
```

Comment:
What are all these parameters? Probably good to describe, or refer to documentation where it is

Reply:
Besides the request/response protocol flows, it will be good to describe, in text, the context (i.e. why/what) of these flows. I can add these specifics in a separate PR.

Reply:
there is a separate doc for all these parameters https://github.com/status-im/react-native-status-keycard/blob/master/docs/usage.md